src/classes/CRLP_RollupCMT.cls

```diff
@@ -302,7 +302,7 @@ public class CRLP_RollupCMT {
                 this.summaryObjectLabel = rlp.Summary_Object__r.Label;
                 this.summaryFieldLabel = rlp.Summary_Field__r.Label;
                 this.detailObjectLabel = rlp.Detail_Object__r.Label;
-                this.detailFieldLabel = rlp.Detail_Field__r.Label;
```

Comment:
Good question as to why one of these used `.Label` and the other used `.MasterLabel`. I thought I had copied these from `CRLP_RollupUI` class, but maybe I did a little of both. That noted, I still see a mix of `MasterLabel` and `Label` used for getting entity reference labels in this class.